client/state/selectors/get-requested-rewind.js

```diff
@@ -0,0 +1,16 @@
+/** @format *
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns an Activity log.
+ *
+ * @param  {Object}        state      Global state tree
+ * @param  {number|string} siteId     the site ID
+ * @return {?Object}                  Activity log item if found, otherwise null
```

Comment:
_Think_ this is a wrong return description. Also is no test for this selector, if you were meaning to add one.

Reply:
Yep, the description was wrong.

I think the value of testing this is limited, but selectors here are supposed to be tested, so… I'll add some.

Reply:
> I think the value of testing this is limited

Yeah, don't feel that you need to add it on my account. I also believe the tests don't add much value for simple selectors. Ironically would have helped in this case because it would have told me the return type :D

Reply:
Let's not do it! Pretty much wind up testing the internals of `lodash.get`.